app/controllers/textings_controller.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+class TextingsController < ApplicationController
+  def create
+    recipient_name = texting_params[:recipient_name]
+    phone_number = parse_phone_number
+    service_id = texting_params[:service_id]
+
+    # Make a request to Textellent API. If request successful we update our DB
+    data = get_db_data(recipient_name, phone_number, service_id)
+    response = post_textellent(data)
+
+    if response['status'] != 'success'
+      puts response['message']
```

Comment:
Nit: Let's not leave in print statements in the code. This ends up clogging the console if everyone just put their prints in the code. If you want to log this, you should use the Rails logger: https://guides.rubyonrails.org/debugging_rails_applications.html#the-logger